synapse/storage/databases/main/state_deltas.py

```diff
@@ -102,20 +105,13 @@ def get_current_state_deltas_txn(txn):
             txn.execute(sql, (prev_stream_id, clipped_stream_id))
             return clipped_stream_id, self.db_pool.cursor_to_dict(txn)
 
-        return self.db_pool.runInteraction(
+        return await self.db_pool.runInteraction(
             "get_current_state_deltas", get_current_state_deltas_txn
         )
 
-    def _get_max_stream_id_in_current_state_deltas_txn(self, txn):
```

Comment:
This gets used in `synapse/storage/databases/main/registration.py:1077`

Reply:
Yeah it gets used in a few places. `synapse/storage/databases/main/state.py:863` also uses it. I reverted this change.